### PR TITLE
feat(ctx): add fallbackFlags support

### LINKED ISSUE #36: Fallback compile flags
There is an option in the LSP called `initializationOptions.fallbackFlags` which allows you to pass some fallback compile flags via the LSP if no infos is found in the `compile_commands.json`.

Would it be possible to enable that?

My problem is that I'm compiling some `.cu` files to some `.ptx` and CMake doesn't register the compilation flags given to `nvcc` for those files into the `compile_commands.json`. Therefore `clangd` raises a bunch of errors due to missing flags (`-std=c++11` among others).

### PR DESCRIPTION
**Pull request by fannheyward**
close #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ This extension connects [coc.nvim][] to the [clangd][] language server.
 - `clangd.disableDiagnostics`: disable diagnostics from clangd, default `false`
 - `clangd.path`: path to `clangd` executable, default `clangd`
 - `clangd.arguments`: arguments for `clangd` server, default `[]`
+- `clangd.fallbackFlags`: flags used when no specific compile commands found, default `[]`
```
Comment (sam-mccall) [inline README.md:35]: suggest "used to parse files" - just to hint that these are clang flags not clangd flags

File: package.json [hunk 2]
```diff
@@ -84,6 +84,14 @@
           },
           "description": "Arguments for clangd server"
         },
+        "clangd.fallbackFlags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [],
+          "description": "Fallback flags used when no specific compile command is found."
```
Comment (sam-mccall) [inline package.json:93]: (and here)

File: README.md (hunk 1 above)
Comment (fannheyward) [reply README.md:35]: You means the description? `flags used to parse files when no specific compile commands found`, is this ok?

Comment (sam-mccall) [reply README.md]: Exactly, thanks!

**Pull request merged**